app/code/Mediotype/Employee/Setup/InstallData.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @author    Mediotype Development <diveinto@mediotype.com>
+ * @copyright 2019 Mediotype. All Rights Reserved.
+ */
+
+namespace Mediotype\Employee\Setup;
+
+use Magento\Framework\Setup\InstallDataInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+
+/**
+ * @codeCoverageIgnore
+ */
+class InstallData implements InstallDataInterface
+{
+    /**
+     * {@inheritdoc}
```

Comment:
```suggestion
     * @inheritdoc
```

Also, we probably don't need all these SuppresWarnings.. I don't think any of them come up for this example